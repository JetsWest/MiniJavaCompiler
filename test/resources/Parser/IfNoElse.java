class M {
    public static void main(String[] args) {
        System.out.println(new B().Test());
    }
}

class B extends A {

    public int Test() {
        int x;
        int y;
        int out;
        x = 2;
        y = 3;
        out = 0;
        if (x < y) {
            out = 1;
        }
        return out;
    }
}