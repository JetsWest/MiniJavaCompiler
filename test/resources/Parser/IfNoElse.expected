Syntax error for input symbol "RETURN" spanning from unknown:19/9(-1) to unknown:19/14(-1)
Couldn't repair and continue parse for input symbol "RETURN" spanning from unknown:19/9(-1) to unknown:19/14(-1)
